.vale.ini

```diff
@@ -4,6 +4,7 @@ Vocab = main
 
 [*.md]
 BasedOnStyles = Microsoft, Vale, write-good, Main
+write-good.Passive = NO
```

Comment:
@darkdrgn2k this is what you suggested right? I think this will clean up most of the Warnings to make things readable.